### ISSUE TITLE
`pat_sitrep()` and git `user.email`

### ISSUE THREAD

**Issue by ijlyttle** [r-lib/usethis#1722]
I'm sure you'll be glad get a break from me, but I found another thing. 

If I have different git `user.email` for `global` and `local`, something like this can happen:

```r
withr::with_envvar(
  list(GITHUB_API_URL = "https://github.<work>.com/api/v3"), 
  git_sitrep(scope = "user")
)
```

```
── Git global (user) 
• Name: 'Ian Lyttle'
• Email: 'ian.lyttle@<work>.com'
• Global (user-level) gitignore file: '~/.gitignore'
• Vaccinated: TRUE
• Default Git protocol: 'https'
• Default initial branch name: 'main'

── GitHub user 
• Default GitHub host: 'https://github.<work>.com'
• Personal access token for 'https://github.<work>.com': '<discovered>'
• GitHub user: 'SESA19001'
• Token scopes: 'gist, repo, user, workflow'
• Email(s): 'ian.lyttle@<work>.com (primary)'
✖ Local Git user's email ('ijlyttle@me.com') doesn't appear to be registered with GitHub host.
```

In this case, `pat_sitrep()` asks for the `"de_facto"` email address, but at the user-level I think it should be asking for the `"global"` email address:

https://github.com/r-lib/usethis/blob/87293f6bcd5dea43420dd4f0763a2a59055d3034/R/github_token.R#L182

I have a fix on my computer where `pat_sitrep()` takes an argument `git_level` with a default `"de_facto"`, and it seems to address the issue.

I could include it with #1721, or make a separate PR - I'll await your signal.

**Comment by jennybc** [comment]
Should `pat_sitrep()` gain `scope = c("user", "project")` as `git_sitrep()` did?

**Comment by ijlyttle** [comment]
For reasons I cannot currently fathom, I opened a duplicate issue: #1732. 😳

It has has a PR that adds the `scope` argument - I'll close the duplicate issue and point that PR here.

**Comment by jennybc** [comment]
Closed in 901dae0542f33697ec89268d52597ecc9e407a9c
